acme/acme/client.py

```diff
@@ -210,23 +210,35 @@ def poll_authorizations(self, orderr: messages.OrderResource, deadline: datetime
             raise errors.ValidationError(failed)
         return orderr.update(authorizations=responses)
 
-    def finalize_order(self, orderr: messages.OrderResource, deadline: datetime.datetime,
-                       fetch_alternative_chains: bool = False) -> messages.OrderResource:
-        """Finalize an order and obtain a certificate.
+    def begin_order_finalization(self, orderr: messages.OrderResource
```

Comment:
It would be good to align this name either with  `poll_finalization` or `finalize_order`. Perhaps:

```suggestion
    def submit_finalization(self, orderr: messages.OrderResource
```